src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -135,9 +144,7 @@ public ReadsSparkSource(final JavaSparkContext ctx, final ValidationStringency v
      * @return RDD of (SAMRecord-backed) GATKReads from the file.
      */
     public JavaRDD<GATKRead> getParallelReads(final String readFileName, final String referencePath, int splitSize) {
-        final SAMFileHeader readsHeader = getHeader(readFileName, referencePath, null);
-        List<SimpleInterval> intervals = IntervalUtils.getAllIntervalsForReference(readsHeader.getSequenceDictionary());
-        return getParallelReads(readFileName, referencePath, intervals, splitSize);
+        return getParallelReads(readFileName, referencePath, null, splitSize);
```

Comment:
Document effect of passing in null, both here and in the javadoc for the four-argument overload. (Eg., is it the same as passing in `IntervalUtils.getAllIntervalsForReference()` as previously, or is the behavior different?